lua/rustaceanvim/lsp.lua

```diff
@@ -96,6 +96,17 @@ local function is_in_workspace(client, root_dir)
   return false
 end
 
+--- Fast fix for root_dir fix on windows
+---@param path string
+---@return string normalize_path
+local function normalize_path(path)
+  local has_windows_drive_letter = path:match '^%a:'
```

Comment:
Sorry, it looks like my other comment got deleted.

I'm not sure about this, but I think we might want to check if we're on Windows here, with `require('rustaceanvim.shell').is_windows()`

Reply:
kk i will use this for avoid unexpected behavior